newsletter/2020-03.md

```diff
@@ -0,0 +1,165 @@
+---
+newsletter:
+  year: 2020
+  month: March
+permalink: /news/2020/03
+---
+
+# AIP Newsletter: March 2020
+
+Welcome to the first (hopefully monthly) edition of the **AIP newsletter**,
+which is designed to keep you up to date about the AIP program, and particular
+proposals making their way through the system.
+
+In this edition, we will discuss five pending proposals, which we intend to
+finalize at the end of March, pending feedback.
```

Comment:
At or by the end of March? If everyone is happy mid-March, can we finalize then?

Reply:
(There are various other occurrences of this - search for "March" to find them.)

Reply:
I think they become approved on March 31 or a similar, pre-specified date (perhaps the last Friday of March instead). The idea behind public comment periods generally is to keep them open for a specified duration of time, because you do not know what additional feedback you might get.

Reply:
I decided to change this to a specific date: March 27. (Depending on how this goes, I will probably always use the last Friday of the month for predictability, and probably skip December entirely.)